src/Guard.php

```diff
@@ -25,13 +32,15 @@ class Guard
      * Create a new guard instance.
      *
      * @param  \Illuminate\Contracts\Auth\Factory  $auth
+     * @param string $provider
      * @param  int  $expiration
      * @return void
      */
-    public function __construct(AuthFactory $auth, $expiration = null)
+    public function __construct(AuthFactory $auth, $provider, $expiration = null)
```

Comment:
We can't do this breaking change on a stable release.

Reply:
Tonight i'll fix that

Reply:
Done